types/logger/rusage_stub.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Tailscale Inc & AUTHORS
 // SPDX-License-Identifier: BSD-3-Clause
 
-//go:build windows || js
+//go:build windows || js || wasip1
```

Comment:
While harmless to leave here, the `js` build tag support in the filch files is being removed.

Reply:
It's one of:

GOARCH=wasm GOOS=js
GOARCH=wasm GOOS=wasip1

So using build tag or filename with `wasm` effectively gets both, as those are the only two geese (GOOSes) with GOARCH=wasm.